test/lit/passes/emit-exnref.wast

```diff
@@ -0,0 +1,28 @@
+;; When given alone, --emit-exnref just runs --translate-to-exnref
```

Comment:
This file has been moved from `experimental-new_eh.wast`. Not sure why it is not recognized as a move.